packages/data/package.json

```diff
@@ -15,5 +15,8 @@
     "@lenster/config": "*",
     "@types/node": "^20.4.4",
     "typescript": "^5.1.6"
+  },
+  "dependencies": {
+    "node-fetch": "^3.3.1"
```

Comment:
By default there is fetch right, no need of this, try removing and run

Reply:
removed the node-fetch pkg and enabled local variable to store the values